includes/embeds/class-amp-facebook-embed.php

```diff
@@ -131,12 +79,20 @@ public function sanitize_raw_embeds( Document $dom ) {
 		 */
 		$fb_root = $dom->getElementById( 'fb-root' );
 		if ( $fb_root ) {
-			$scripts = [];
-			foreach ( $dom->xpath->query( '//script[ starts-with( @src, "https://connect.facebook.net" ) and contains( @src, "sdk.js" ) ]' ) as $script ) {
-				$scripts[] = $script;
-			}
-			foreach ( $scripts as $script ) {
-				$script->parentNode->removeChild( $script );
+			/**
+			 * Script.
+			 *
+			 * @var DOMElement $script
+			 */
+			$script_query = $dom->xpath->query( '//script[ starts-with( @src, "https://connect.facebook.net" ) and contains( @src, "sdk.js" ) ]' );
+			foreach ( $script_query as $script ) {
+				$parent_node = $script->parentNode;
+				$parent_node->removeChild( $script );
+
+				// Remove parent node if it is an empty <p> tag.
+				if ( 'p' === $parent_node->nodeName && null === $parent_node->firstChild ) {
+					$parent_node->parentNode->removeChild( $parent_node );
+				}
```

Comment:
I don't see a test for this.

Reply:
Done in https://github.com/ampproject/amp-wp/pull/4384/commits/c6869a9380c7842ed0f176bff2a0579012fff3a0.